### ISSUE TITLE
Can't distinguish between "2012" and "2012 R2"

### ISSUE THREAD

**Issue by DavidS** [mcanevet/rspec-puppet-facts#55]
With either "2012" or "2012 R2" as operatingsystemrelease in metadata.json, unit tests are always run for both entries:

```
david@davids:~/tmp/testmod$ grep -A3 windows metadata.json 
      "operatingsystem": "windows",
      "operatingsystemrelease": [
        "2012 R2"
      ]
david@davids:~/tmp/testmod$ bundle exec rspec -fd spec/classes/foo_spec.rb 

testmod::foo
  on debian-8-x86_64
    should compile into a catalogue without dependency cycles (FAILED - 1)
  on windows-2012-x64
    should compile into a catalogue without dependency cycles (FAILED - 2)
  on windows-2012 R2-x64
    should compile into a catalogue without dependency cycles (FAILED - 3)
  on ubuntu-16.04-x86_64
    should compile into a catalogue without dependency cycles (FAILED - 4)
```

**Comment by rodjek** [comment]
This will be resolved by #56

**Comment by DavidS** [comment]
Fixed through #56.
